staging/istio/files/profile-ambient.yaml

```diff
@@ -0,0 +1,25 @@
+# The ambient profile enables ambient mode. The Istiod, CNI, and ztunnel charts must be deployed
```

Comment:
@gracedo If this file is auto-generated or it is based on opensource istio?

Reply:
it is directly taken from upstream/opensource istio helm chart